src/main/resources/avro/referencemethods.avdl

```diff
@@ -12,19 +12,16 @@ as JSON.
 */
 record SearchReferenceSetsRequest {
   /**
-  If nonempty, return the reference sets which match any of the given
-  `md5checksum`s. See `ReferenceSet::md5checksum` for details.
+  If not null, return the reference sets which match the given
+  `md5checksum`. See `ReferenceSet::md5checksum` for details.
   */
-  array<string> md5checksums = [];
+  union { null, string } md5checksum = null;
 
   /**
-  If nonempty, return reference sets for which the accession
-  matches this string. Best to give a version number (e.g. `GCF_000001405.26`).
-  If only the main accession number is given then all records with
-  that main accession will be returned, whichever version.
-  Note that different versions will have different sequences.
+  If not null, return reference sets for which the accession
+  matches this string.
```

Comment:
Does "matches this string" mean "is equal to this string" or "contains this substring" ? (I think I know the intent, but given that we're using the same phrase before and after this PR, I'm not sure.)